### ISSUE TITLE
Add Record Option / Tie in PS Vue MyDVR to PVR Recordings

### ISSUE THREAD

**Issue by eracknaphobia** [eracknaphobia/service.psvue.epg#23]
Not sure if this one's even possible, but it would be awesome.

**Comment by DiR3Wolf** [comment]
I think this should be possible through skinning, but will need skin developer support. It is a good idea. I will be developing a skin (for personal use and may take a few months), and if I get this working, I should be able to help with a mod file for others skins.

**Comment by DiR3Wolf** [comment]
I don't think that is possible for only one episode. I think DVR feature on PS Vue only works with all episodes the deem ok to be available. At least that is how it works on Apple TV. Some episodes/movies disappear if they lapse (i think 30 days), but it the network runs them again, they will reappear.
